app/src/containers/CartPage.tsx

```diff
@@ -86,34 +129,59 @@ class CartPage extends React.Component<RouteComponentProps, CartPageState> {
 
   fetchCartData() {
     login().then(() => {
-      cortexFetch(`/?zoom=${zoomArray.sort().join()}`, {
+      cortexFetch('/', {
```

Comment:
We could save on the overhead of making two requests by making a single as before, and update the zoom query to request both `carts` and `defaultCart`. Then update the component state based on what's returned in the response.